modelci/cli.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Author: Li Yuanming
+Email: yli056@e.ntu.edu.sg
+Date: 10/12/2020
+"""
+import click
+
+import modelci.hub.cli
```

Comment:
from hub import cli is weird
we should have a CLI directory.
we'd better use a hub_cli.py in a CLI directory.

reorgainize this

Reply:
This hierarchy is from mlflow. They have sperate cli.py under each module.
For example:
CLI entry point `cli.py`:
https://github.com/mlflow/mlflow/blob/master/mlflow/cli.py
CLI for SageMaker under `sagemaker/cli.py`:
https://github.com/mlflow/mlflow/blob/master/mlflow/sagemaker/cli.py

Reply:
refactored